client/descriptions.go

```diff
@@ -287,6 +287,24 @@ func (f FieldDescription) IsPrimaryRelation() bool {
 	return f.RelationType > 0 && f.RelationType&Relation_Type_Primary != 0
 }
 
+// IsRelation returns true if this field is a relation.
+func (f FieldDescription) IsRelation() bool {
+	return f.RelationType > 0
+}
+
+// IsArray returns true if this field is an array type.
```

Comment:
todo: Looking at the implementation, it looks like this function will return true if it is an inline array, or the many side of a one-many.  I think it is really worth highlighting this in the documentation as it would be easy enough to assume otherwise.

Reply:
I'm not quite following. What do you mean by "inline array"? And it's not quite clear what can be assumed otherwise. It definitely can't be the one side of one-many.

Reply:
An inline array is an array defined on the object itself (not a relation), i.e. `FieldKind_INT_ARRAY` or `FieldKind_BOOL_ARRAY`.

Pretty much everywhere in our code base differentiates (often out of out of technical necessity) between inline arrays and relation-arrays.  It is strange and unexpected to find a function that does not, and as such adding a handful of words to the documentation to make this clear will likely save a headache or too in the future.

Reply:
done